src/SFA.DAS.ApprenticeCommitments.Jobs.Functions/Infrastructure/Startup.cs

```diff
@@ -19,7 +21,18 @@ public override void ConfigureAppConfiguration(IFunctionsConfigurationBuilder bu
         public override void Configure(IFunctionsHostBuilder builder)
         {
             builder.ConfigureLogging();
-            builder.ConfigureNServiceBus();
+
+            builder.UseNServiceBus(() =>
+            {
+                var configuration = new ServiceBusTriggeredEndpointConfiguration(
+                    endpointName: QueueNames.ApprenticeshipCreatedEvent,
+                    connectionStringName: "NServiceBusConnectionString");
+
+                configuration.DoNotSendMessagesToErrorQueue();
+                configuration.LogDiagnostics();
+
```

Comment:
This is Azure ServiceBus specific functionality for NSB - it tells NSB to put failed messages in the ServiceBus dead letter queue.  Without explicitly stating it, messages appear to be dropped otherwise!

Reply:
just an interesting name 'DoNotSendMessagesToErrorQueue' appear to imply we don't send to error queue

Reply:
Yes, it does seem non-intuitive.  I think it's attempting to distinguish from the usual NServicebus style error queues, which are really just another queue with "-error" tacked onto the name.

Reply:
yeah naming things is always a pain